docs/tutorials/reuse-ssh-connections.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Reuse SSH Connections
+group: advanced-networking
+author: [ "@Dave-Baum" ]
+---
+
+* Table of Contents
+{:toc}
+
+## Intro
+
+Establishing an ssh connection requires a a lot of communication between your host computer
+and the EV3 and can take several seconds.  Although this is fine a long-lived connection,
+it can be quite tedious if you want to use ssh to invoke a single command or scp to transfer
+files.  By instructing ssh to reuse connections, each connection after the first one will
+reuse the session that was already established, drastically speeding up ssh and scp commands.
+
+## Congifuration (OSX, Linux)
+
+Edit or create a new file at `~/.ssh/config`.  Add the following contents:
+
+    Host <name>
+      HostName <address>
+      User <user>
+      ControlMaster auto
+      ControlPath ~/.ssh/master-%r@%h:%p
+
```

Comment:
Is the extra indentation meaningful for the lines after the first? I don't see anything in the `ssh_config` man page about this.